src/triangle.jl

```diff
@@ -3,57 +3,174 @@ triangle.jl
 2D triangle shape
 =#
 
-using ImagePhantoms #: Object2d
+
+using ImagePhantoms #: Object, Object2d
 
 export Triangle
+export phantom, radon, spectrum
+
+const sqrt3 = sqrt(3)
```

Comment:
If this is for performance consideration, then it is not necessary; the Julia compiler would just optimize these constant operations.

Reply:
It's mostly to save typing :)